examples/sv3d/simple_video_sample.py

```diff
@@ -0,0 +1,325 @@
+""" Note that this file is adapted from `simple_video_sample.py`, as modular design is current not supported under examples.
+
+Adapted from https://github.com/Stability-AI/generative-models/blob/main/scripts/sampling/simple_video_sample.py
+python simple_video_sample.py --version sv3d_u
+"""
+
+from __future__ import annotations
+
+import argparse
+import math
+import os
+import sys
+from glob import glob
+from pathlib import Path
+from typing import Optional
+
+from loguru import logger
+
+__dir__ = os.path.dirname(os.path.abspath(__file__))
+mindone_lib_path = os.path.abspath(os.path.join(__dir__, "../../"))
+sys.path.insert(0, mindone_lib_path)
+
+import cv2
+import imageio
+import numpy as np
+from einops import rearrange
+from omegaconf import OmegaConf
+from PIL import Image
+from rembg import remove
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # FIXME: python 3.7
+
+from sgm.helpers import create_model_sv3d as create_model
+from sgm.util import default
+from utils import mixed_precision
+
+import mindspore as ms
+from mindspore import Tensor, nn, ops
+from mindspore.dataset.vision import ToTensor
+
+from mindone.utils.seed import set_random_seed
+
+
+class SVD3InferPipeline(nn.Cell):
```

Comment:
`SV3DInferPipeline`?

Reply:
> `SV3DInferPipeline`?

Hi Rustam thank you so much for the reviewing efforts. The typo has been fixed thanks.